lib/pengine/bundle.c

```diff
@@ -1589,6 +1589,16 @@ pe__bundle_xml(pcmk__output_t *out, va_list args)
     }
 
     if (printed_header) {
+        if (pcmk_is_set(options, pe_print_xml)) {
+            char *rsc_xml = dump_xml_formatted(rsc->orig_xml ? rsc->orig_xml : rsc->xml);
+            out->output_xml(out, "raw-xml", rsc_xml);
+            free(rsc_xml);
+        } else if (pcmk_is_set(options, pe_print_expanded_xml)) {
+            char *rsc_xml = dump_xml_formatted(rsc->xml);
+            out->output_xml(out, "xml", rsc_xml);
+            free(rsc_xml);
+        }
```

Comment:
If we're going to have a separate message just for printing out the resource config, then perhaps we don't need f22783292fbfecbf2d8ff62c795160190f6fa2d2.  I only added that patch to support putting this code into the messages.  And then if that's the case, perhaps we should go ahead and revert it immediately.  It adds to pe_print_options (which looks public to me), but it's only been released in -rc1.  I think it would be okay for us to make that change.

Reply:
That does sound good. We can keep that commit's formatting changes and just drop the new flag